app/controllers/deploys_controller.rb

```diff
@@ -94,6 +94,7 @@ def search
 
   def new
     @deploy = current_project.deploys.build(params.except(:project_id).permit(:stage_id, :reference))
+    @recent_releases = current_project.releases.order(id: :desc).last(5)
```

Comment:
oh oops, should be `.first(5)`. I just wanted to show the most recent release first.